src/org/zaproxy/zap/extension/websocket/ExtensionWebSocket.java

```diff
@@ -950,8 +943,8 @@ public WebSocketMessageDTO getWebsocketMessage(int messageId, int channelId) thr
 	 * @return the specified messages
 	 * @throws DatabaseException
 	 */
-	public List<WebSocketMessageDTO> getWebsocketMessages(WebSocketMessageDTO criteria, List<Integer> opcodes, List<Integer> inScopeChannelIds, int offset, int limit, int payloadPreviewLength) throws DatabaseException {
-		return this.table.getMessages(criteria, opcodes, inScopeChannelIds, offset, limit, payloadPreviewLength);
+	public List<WebSocketMessageDTO> getWebsocketMessages(WebSocketMessageDTO criteria, List<Integer> opcodes, List<Integer> inScopeChannelIds, WebSocketMessagesViewPayloadFilterVo payloadFilterVo, int offset, int limit, int payloadPreviewLength) throws DatabaseException {
```

Comment:
I am going to remove `Vo`  and `View`(because that class doesn't related with ui) from class name. Do you think is that ok? In addition, I am not sure if websocket.ui is the appropriate package. Maybe I should move the class to websocket package